plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/GodotPlugin.kt

```diff
@@ -82,13 +81,13 @@ class GodotPlugin : Plugin<Project> {
             val buildTask = project.tasks.getByName("build")
 
             // clean up task
-            if (godot.cleanupGeneratedFiles.get()) {
-                val cleanGeneratedFilesTask = project.tasks.register<Delete>("cleanGeneratedFiles") {
-                    group = GODOT_TASK_GROUP
-                    delete(godot.gdnsDir.get())
-                }
-                buildTask.dependsOn(cleanGeneratedFilesTask)
-            }
+//            if (godot.cleanupGeneratedFiles.get()) {
+//                val cleanGeneratedFilesTask = project.tasks.register<Delete>("cleanGeneratedFiles") {
+//                    group = GODOT_TASK_GROUP
+//                    delete(godot.gdnsDir.get())
+//                }
+//                buildTask.dependsOn(cleanGeneratedFilesTask)
+//            }
```

Comment:
Commented as discussed on discord

Reply:
Add a comment why? So we don't forget :)

Reply:
:+1: